lib/data_cycle/check_course_jobs_logging.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_dependency "#{Rails.root}/lib/data_cycle/update_logger"
+
+module CheckCourseJobsLogging
+  def log_orphan_record
+    Raven.capture_message('Orphan lock removed',
+                          level: 'warn',
+                          extra: sentry_extra)
+    return nil
+  end
+
+  def log_previous_failed_update
+    UpdateLogger.update_course(@course, 'orphan_lock_failure' => true)
```

Comment:
I think it would be better to keep the timestamp for this. It could just be the timestamp instead of `true`.

Reply:
Done!